### DISCUSSION TITLE
Translation to more languages

### DISCUSSION THREAD

**Discussion by madjesse** [ant-design/ant-design#30785] [General]
I am a developer based in Japan and I find antd extremely powerful.
I just wonder if there is any ongoing plan for translating the project to Japanese so that we can grow a community here?
This applies to communities of other languages as well, I think.

**Comment by yoyo837** [comment]
What are the details of the plan?

**Reply by madjesse** [reply]
Thank you.
I have not yet worked out the details but can we discuss it?
How did you do it for the existing language versions like English and Portuguese? I would be happy if you could share some of your know-how in doing this.
(p.s. 我说中文)

**Reply by yoyo837** [reply]
所有翻译都是社区贡献的，如果您想翻译其他的，请给直接我们发 PR，并最好协助保持同步。

**Comment by EdiJunior88** [comment]
I am a Brazilian developer and would like to contribute to the translation into Brazilian Portuguese (currently it only has translation from English and Chinese)

**Reply by yoyo837** [reply]
There was a time when we had readme.md files in multiple languages. Later, the team found that it was very difficult to maintain documentation in multiple languages. We did not master these languages, and the maintenance of multilingual documentation eventually became unmaintained and out of sync, so we deleted all documentation except English and Chinese.
